.github/ISSUE_TEMPLATE/linkcheck_report.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 🐛 Weekly linkcheck failed
+about: Use this template for checking broken links
+name: Weekly linkcheck failed
+labels: Maintanance, bug
```

Comment:
```suggestion
labels: maintanance, bug
```

Reply:
There is a small typo so I had to add in a new commit.